pkg/backend/apply.go

```diff
@@ -236,7 +236,20 @@ func createDiff(updateKind apitype.UpdateKind, events []engine.Event, displayOpt
 	seen := make(map[resource.URN]engine.StepEventMetadata)
 	displayOpts.SummaryDiff = true
 
+	outputEvents := make([]engine.Event, 0)
 	for _, e := range events {
+		// display output events last
+		if e.Type == engine.ResourceOutputsEvent {
+			outputEvents = append(outputEvents, e)
+			continue
+		}
+		msg := display.RenderDiffEvent(e, seen, displayOpts)
+		if msg != "" && e.Type != engine.SummaryEvent {
+			_, err := buff.WriteString(msg)
+			contract.IgnoreError(err)
+		}
```

Comment:
It looks like this code is also duplicated below, but won't be run as the only `outputEvents` are `engine.ResourceOutputEvents`.

is it possible to remove the bottom code?

```go
		if msg != "" && e.Type != engine.SummaryEvent {
			_, err := buff.WriteString(msg)
			contract.IgnoreError(err)
		}
```

Reply:
Good catch 😅